packages/core/src/internal/execution/types/execution-result.ts

```diff
@@ -59,6 +60,16 @@ export interface StrategyErrorExecutionResult {
   error: string;
 }
 
+/**
+ * The execution strategy returned a strategy-specific hold e.g.
+ * waiting for off-chain multi-sig confirmations.
+ */
+export interface StrategyHeldExecutionResult {
```

Comment:
The appropriate (failable) exstates can return a complete message where this Hold is the result.